src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestDetailsView.tsx

```diff
@@ -105,27 +133,103 @@ const ChangeRequestDetailsView: React.FC<ChangeRequestDetailsProps> = ({
+      return (
+        <div>
+          <NERButton
+            endIcon={<ArrowDropDownIcon style={{ fontSize: 28 }} />}
+            variant="contained"
+            id="unreviewed-cr-actions-dropdown"
+            onClick={handleClick}
+          >
+            Actions
+          </NERButton>
+          <Menu open={dropdownOpen} anchorEl={anchorEl} onClose={handleDropdownClose}>
+            <MenuItem onClick={handleReviewOpen} disabled={!isUserAllowedToReview}>
+              Review
+            </MenuItem>
+            <Divider />
+            <MenuItem disabled={!isUserAllowedToDelete} onClick={handleDeleteOpen}>
+              Delete
+            </MenuItem>
+          </Menu>
+          <Grid container spacing={2}>
+            <Grid item xs={12} md={8}>
+              <Autocomplete
+                isOptionEqualToValue={(option, value) => option.id === value.id}
+                limitTags={1}
+                disableCloseOnSelect
+                multiple
+                options={users.filter((user) => isLeadership(user.role)).map(userToAutocompleteOption)}
+                getOptionLabel={(option) => option.label}
+                onChange={(_, values) => setReviewerIds(values.map((value) => value.id))}
+                renderOption={(props, option, { selected }) => (
+                  <li {...props}>
+                    <Checkbox
+                      icon={<CheckBoxOutlineBlankIcon />}
+                      checkedIcon={<CheckBoxIcon />}
+                      style={{ marginRight: 8 }}
+                      checked={selected}
+                    />
+                    {option.label}
+                  </li>
+                )}
+                renderTags={() => null}
+                renderInput={(params) => (
+                  <TextField {...params} variant="standard" label="Add Reviewers" placeholder="Select Reviewers" />
+                )}
+              />
+            </Grid>
+            <Grid item xs={12} md={4}>
+              <NERButton
+                sx={{ mt: '20px', float: 'right' }}
+                variant="contained"
+                disabled={!isRequestAllowed}
+                onClick={handleRequestReviewerClick}
+              >
+                Request Review
+              </NERButton>
+            </Grid>
+          </Grid>
+        </div>
+      );
+    }
+    return (
+      <div>
+        <NERButton
+          endIcon={<ArrowDropDownIcon style={{ fontSize: 28 }} />}
+          variant="contained"
+          id="unreviewed-cr-actions-dropdown"
+          onClick={handleClick}
+        >
+          Actions
+        </NERButton>
+        <Menu open={dropdownOpen} anchorEl={anchorEl} onClose={handleDropdownClose}>
+          <MenuItem onClick={handleReviewOpen} disabled={!isUserAllowedToReview}>
+            Review
+          </MenuItem>
+          <Divider />
+          <MenuItem disabled={!isUserAllowedToDelete} onClick={handleDeleteOpen}>
+            Delete
+          </MenuItem>
+        </Menu>
```

Comment:
when you abstract this action button component, use the ActionsMenu component that Peyton made, you will need to modify it so you can add the divider between review and delete